### DISCUSSION TITLE
Go Pro Labs extra-long photo timelapse + scheduled start

### DISCUSSION THREAD

**Discussion by JeremyKarlDay** [gopro/labs#524] [Q&A] [answered]
I work in ecological research and am setting up a new experiment, which I want to use 4 x GoPro black 11’s and some torches to monitor overnight. I installed the GoPro labs firmware, and reckon it is terrific. Here is my query;

GoPro labs has an extra long Timelapse photo setting which I need to use to monitor the experiment. I have run some tests, and I get ~8 hrs of battery life at 30 seconds between photos. This is exactly what I was looking for.

However, I also would like to schedule the cameras to turn on at a certain time, say 7pm, as this would be a big quality of life adjustment for me. 

However, I can't see the option to have both the extra-long photo timelapse, which turns the camera off between photos (this is the only way the battery life will last long enough) AND schedule the camera to switch on at 7pm. I know I can create custom QR codes, but I haven't found the code for 'extra-long photo timelapse', only 'photo timelapse' and this does not turn the cameras off between photos, leading to only 2hrs battery life. 

I wonder if someone here could tell me where I'm going wrong, or whether the settings I seek are possible on a GoPro black 11. Many thanks in advance for any assistance,

JD

**Comment by JeremyKarlDay** [comment]
This issue has been resolved. For anyone else frantically looking for how to do this in future, you can alter the code directly. I have learned something today. The QR code I generated to achieve the outcome I wanted is shown below. Great product. JD

https://gopro.github.io/labs/control/set/?cmd=!18:00NmTPfW0g0v0q0dToV4B1D0R1C1S1oT230911203548!S!06:00E!56NQmPN!S!1R

**Reply by dnewman-gpsw** [reply, marked as answer]
Almost there.  Yes you are understanding how to combine the features well, the one feature you should not combine is setting time within a QR code that uses a repeat (!R) command.  Each time the command repeats, it will set the time and data wrong.

**Reply by JeremyKarlDay** [reply]
You legend! Thanks very much - you’ve definitely saved me some heart ache there. JD
